README.md

```diff
@@ -12,21 +12,29 @@
 
 This is the Rust implementation of the dotLottie player and its related tools. It utilizes uniffi-rs to generate FFI bindings for Kotlin, Swift, and WebAssembly (WASM). these bindings are then used in the native dotLottie players for [Android](https://github.com/LottieFiles/dotlottie-android), [iOS](https://github.com/LottieFiles/dotlottie-ios), and [Web](https://github.com/LottieFiles/dotlottie-web) bringing consistency of playback and dotLottie features across all platforms.
```

Comment:
Should we include:

- The problem dotlottie-rs solves (fragmentation and rendering inconsistency issues)
- The features it supports

Reply:
added @theashraf please review